### DISCUSSION TITLE
Ampache 5.2.0 - albums broken up by year??

### DISCUSSION THREAD

**Discussion by JoeHartley3** [ampache/ampache#3165] [Q&A] [answered]
Hi, all.  I've been using Ampache for years, and have been able to extend my usage over the years to be able to integrate with VolumeIO and my phone's Subsonic client.  Cool stuff!  But here's my problem:

I have a number of albums that have songs from multiple years - an example is NRBQ's "Peek-a-boo: The Best Of NRBQ (1969-1989)".

When I search for this in Ampache, it comes up as TWELVE different albums, one for each year that the songs on it were originally released.  That is, I'll search for the album by name, and it comes up with twelve entries, all with the same album name, but with different years shown, and only the songs released in that year are included in that "sub-album".

This is REALLY annoying, but I have not been able to work out how to have the catalog (or is it the search?) return only the one album rather than the dozen Ampache thinks it is.

I'd be grateful if anyone has a clue how to make this work the way I'd like it to.  Thanks!

**Comment by mitchray** [comment]
Can you check if the album has a year set?

If it doesn't then it may be looking for the year from songs, OR its overriding the album year with song year

**Reply by JoeHartley3** [reply]
Each instance of the album has a different year.  In diving into the metadata, each song had its original year of release in the metadata.  I hadn't done the original rip, and the metadata said it had been done by iTunes, which I've never used, so I re-ripped the CDs (which gave me a chance to upgrade the files to Ogg Vorbis anyway).

The new files did not have the original year in the song's metadata, and after updating the catalog, each disc shows in the catalog once, and has all the files I'd expect.

So my question is redefined, I suppose: how can I get Ampache to ignore the original year in the metadata?  And is this something that happens at cataloging or when displaying a search?

Thanks for the pointer, Mitch.

**Comment by mitchray** [comment, marked as answer]
Short answer is to just have a single 'year' tag for all the songs, which correlates to the release year of the album itself. 
https://github.com/ampache/ampache/issues/2326#issuecomment-594841446

Having a different 'Original Year' in any of the tracks will splinter the album as you've experienced
https://github.com/ampache/ampache/issues/3018#issuecomment-979502256
